app/views/admin/_metadata_summary_card.html.erb

```diff
@@ -2,45 +2,52 @@
   summary_card_actions ||= []
 %>
 
+<% related_links_value = capture do %>
+  <% if schema["related"] %>
+    Yes
+    <% schema["related"].each_with_index do |content_id, index| %>
+      <p>Link <%= index + 1 %>: <%= content_id %></p>
```

Comment:
Could we have a screenshot of this (more complicated) outcome, rather than the "No" outcome you have in the PR description currently?